assets/js/system-status.ts

```diff
@@ -7,65 +7,54 @@ window.addEventListener('DOMContentLoaded', () => {
   );
 
   const $operationalIcons = document.querySelectorAll('.operational');
-  const $degradedPerformanceIcons = document.querySelectorAll(
-    '.degraded-performance'
-  );
-  const $partialServiceDisruptionIcons = document.querySelectorAll(
-    '.partial-service-disruption'
-  );
-  const $plannedMaintenanceIcons = document.querySelectorAll(
-    '.planned-maintenance'
-  );
-  const $serviceDisruptionIcons = document.querySelectorAll(
-    '.service-disruption'
-  );
-  const $securityEventIcons = document.querySelectorAll('.security-event');
+  const $warningIcons = document.querySelectorAll('.degraded-performance');
   const $allSystemStatusIcons = document.querySelectorAll('.status-icon');
 
   type SystemStatus = {
-    icons: NodeList;
+    icon: NodeList;
```

Comment:
Nitpick, but I think `icons` makes more sense since it is a list of icons (one in the header and one in the footer).